pkg/apis/externaldns/types.go

```diff
@@ -142,6 +144,8 @@ var defaultConfig = &Config{
 	ExoscaleEndpoint:         "https://api.exoscale.ch/dns",
 	ExoscaleAPIKey:           "",
 	ExoscaleAPISecret:        "",
+	CRDSourceAPIVersion:      "",
```

Comment:
what do you think of externaldns.k8s.io/v1alpha  as APIVersion and DNSEndpoint  as Kind for default settings?

Reply:
Agree. Done